packages/l/libmem/port/xmake.lua

```diff
@@ -0,0 +1,132 @@
+add_rules("mode.debug", "mode.release")
+set_languages("c17", "c++20")
+add_headerfiles("include/(libmem/**.h)")
+add_headerfiles("include/(libmem/**.hpp)")
+if is_plat("windows") then
+    set_toolset("make", "nmake") -- Use NMAKE as the make tool
+end
+
+-- Set options
+option("build_tests", {description = "Build tests", default = false, showmenu = true})
+option("deep_tests", {description = "Enable deep testing", default = false, showmenu = true})
```

Comment:
we need not tests options.

Reply:
Done Ive removed all options related to tests.